### ISSUE TITLE
various problems with atom colors

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/states-of-matter#40]
(1) The sim does not use color-blind red for oxygen, 'rgb(255,85,0)'.

(2) In fact, it uses 2 different colors for oxygen in SolidLiquidGasMoleculesControlPanel:
> 154 var dot1 = new Circle( 5, { fill: '#D91200' } );
> 166 var dot1 = new Circle( 5, { fill: '#DA1300' } );
> 167 var dot2 = new Circle( 5, { fill: '#DA1300', left: dot1.right - 4 } );

(3) Color are not encapsulated anywhere. Changing the colors in SolidLiquidGasMoleculesControlPanel (for example) does not change them elsewhere. This indicates that color literals are duplicated, making them difficult to change.

**Comment by ChandrashekarBemagoni** [comment]
Fixed in above commit .Assinging to @pixelzoom for review.

**Comment by pixelzoom** [comment]
StatesOfMatterConstants.OXYGEN_COLOR was not the correct value for color-blind red. I've changed it. Assigning to @jbphet for approval.

**Comment by jbphet** [comment]
@arouinfar is going to use the colors from the original simulation.  She inherited the spec, and doesn't know why the colors were changed, so we're going to change them back since they were vetted for colorblindness back when the Java version was released.

**Comment by pixelzoom** [comment]
As mentioned in https://github.com/phetsims/states-of-matter/issues/50, colors in nitroglycerin have also been approved for use across all chemistry sims. If this sim is going to deviate from those colors, that should be part of a large 'chem team' discussion.

**Comment by ycarpenterphet** [comment]
@pixelzoom Do you know if the colors described in nitroglycerin cover Neon and Argon atoms? 

I can see sticking with consistent colors vs. nitroglycerin for the oxygen atoms in oxygen and water here, but we may have to play around with the colour of the large moveable atom in the Interactions screen and in Atomic Interactions.

**Comment by pixelzoom** [comment]
As noted in https://github.com/phetsims/states-of-matter/issues/50, nitroglycerin does not currently include Neon and Argon. And it would be nice if chem sims would proactively contribute new things to nitroglycerin that other sims can reuse, rather than waiting for someone to later discover that a sim already defined (possibly different) properties for an Element.

**Comment by ycarpenterphet** [comment]
I suspect the reason that Ne and Ar are not part of nitroglycerin is that they don't have standard colors in chemistry -- whereas commonly discussed atoms like H, O, N do have disciplinary conventions.

It would be useful for our sims to be consistent between sims where possible, though. 

Either way, @arouinfar and I can discuss the color used for oxygen in the SOM suite and decide how to proceed on this (i.e. if we can tweak the moveable atom to match the existing nitroglycerin specified color for oxygen, or if this will create other problems specific to this sim).

For argon and neon, we should use the Java sim colors.

**Comment by jbphet** [comment]
Neon and Argon were added to the nitroglycerin repo a while back, see https://github.com/phetsims/nitroglycerin/commit/826a0d7c848bf9271ef37950963ce7c3fb082f28.  The colors for all atoms are now drawn from nitroglycerin.  We did a little bit of updating to the color used for neon in order to make it stand out a bit more, see https://github.com/phetsims/states-of-matter/issues/49.  I think our work here is done.  Assigning to @arouinfar to verify, she can close if all looks good.  The colors found in this version are up to date: http://www.colorado.edu/physics/phet/dev/html/states-of-matter/1.0.0-dev.7/states-of-matter_en.html.
